superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/getControlItemsMap.tsx

```diff
@@ -214,6 +216,69 @@ export default function getControlItemsMap({
       );
       mapControlItems[controlItem.name] = { element, checked: initialValue };
     });
+  controlItems
+    .filter(
+      (controlItem: CustomControlItem) =>
+        controlItem?.config?.renderTrigger &&
+        controlItem.name !== 'sortAscending' &&
```

Comment:
We probably don't need this check?

Reply:
You mean and just leave it to checking if it's a SelectControl? I'm fine with that but honestly I never checked what those other conditions are for. I just duplicated the previous section and tweaked it for a select :laughing:

Reply:
renderTrigger is needed, otherwise the COLUMN select box shows up again. sortAscending isn't though.